recipes/linecache2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "linecache2" %}
+{%set version = "1.0.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4b26ff4e7110db76eeb6f5a7b64a82623839d595c2038eeda662f2a2db78e97c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
```

Comment:
My patches aren't extending the suite - I'm explicitly commenting out the tests that caused build issues when byte-compiling for python 2. As such, I consider this more of a reduction in function than an expansion.